api/services/tasks/v1/tasks.proto

```diff
@@ -72,6 +72,8 @@ message CreateTaskRequest {
 	bool terminal = 7;
 
 	containerd.types.Descriptor checkpoint = 8;
+
+	google.protobuf.Any options = 9;
```

Comment:
Do we need to remove the options field from the Container object? Right now, we have a `map<string, string>` there as well.